package.json

```diff
@@ -9,6 +9,7 @@
   "license": "MIT",
   "private": false,
   "scripts": {
+    "test": "yarn lint && jest",
```

Comment:
Are you sure, that `test` command should do something instead of running tests?

Reply:
This `test` command runs the tests.